### DISCUSSION TITLE
State of the Project

### DISCUSSION THREAD

**Discussion by mozman** [mozman/ezdxf#1175] [Announcements]
I would like to keep you up to date on the current development of `ezdxf` .

My current motivation to work on `ezdxf` is unfortunately very low. Please understand that I currently have no capacity for new developments.

But you can rely on `ezdxf` remaining stable and functional. I will make sure that `ezdxf` is compatible with future Python versions and that existing functions work reliably.

To ensure the stability of `ezdxf`, I am currently concentrating on fixing bugs and ensuring compatibility with future Python versions. Changes that go beyond bug fixing are currently not being integrated into the project. This is because I do not consider the test environment to be comprehensive enough to safely introduce changes without having to invest some time, which I do not want to do.

I will keep you updated as soon as the situation changes and I have more motivation to work on `ezdxf` again.

Best regards,
Manfred

**Comment by ghost** [comment]
Hey Manfred,
I see where you're coming from.
We rely on the ezdxf project in our company. Please let us know if you receive sponsorships: I personally recommend checking out [Tidelift](https://tidelift.com/) and signing up as a maintainer. I'd be glad to answer any questions you can have—if so, feel free to reach me out on Discord, `@bswck`.

Best,
Bartosz (@bswck)

**Comment by johnthagen** [comment]
It may be worth converting this to a pinned issue for greater visibility.

- https://docs.github.com/en/issues/tracking-your-work-with-issues/administering-issues/pinning-an-issue-to-your-repository

This would also increase visibility for possible sponsorships / support.

Edit: 

I looks like you can pin a discussion too, but issues are still probably more visible for most users?

- https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#pinning-a-discussion

**Reply by Pat-Ser** [reply]
Hello Manfred,
A simple but big "thank you" for the great job you've done.
Sincerly
